recipe/meta.yaml

```diff
@@ -36,7 +36,7 @@ outputs:
     script: install_base.bat  # [win]
     build:
       entry_points:
-        - f2py = numpy.f2py.f2py2e:main
+        - f2py = numpy.f2py.f2py2e:main  # [win]
```

Comment:
Does this entry point cause failure on UNIX machines?

Reply:
I believe so. I made this change working on a docker image running it on my local machine. Let me test and make sure it is necessary still.

Reply:
This caused a failure for the windows machine if I remember correctly. I went ahead and changed it without the `win` selector to double-check and see for a failure.